crates/node/src/commands/run.rs

```diff
@@ -26,6 +28,11 @@ pub struct ExtendedRunCmd {
     #[clap(long, value_enum, ignore_case = true)]
     pub sealing: Option<Sealing>,
 
+    // TODO. Move this into `Sealing::Instant` when https://github.com/clap-rs/clap/issues/2621 is resolved.
+    /// Finalize sealed blocks. This flag is ignored if sealing is set to `Manual`.
+    #[clap(long, default_value = "false", default_missing_value = "true")]
```

Comment:
```
        // This combination of properties allow us to use a default value only if the arg is passed without value.
        // If it is not passed at all, no default value is used.
        // See: https://docs.rs/clap/latest/clap/struct.Arg.html#method.default_missing_value
        num_args = 0..=1, require_equals=true, default_missing_value = Some(constants::DEFAULT_CONFIGS_URL)
```
you have to provide all three of the arguments if you want to use `default_missing_value` correctly

Reply:
yes, you are right. It is not needed for `bool` at all